Source/Diagnostics/ReducedDiags/ColliderRelevant.H

```diff
@@ -0,0 +1,61 @@
+/* Copyright 2023 The WarpX Community
+ *
+ * This file is part of WarpX.
+ *
+ * Authors: Yinjian Zhao, Arianna Formenti
```

Comment:
Unique enough on this file :)
```suggestion
 * Authors: Arianna Formenti
```